be/src/vec/sink/load_stream_stub.cpp

```diff
@@ -108,6 +108,10 @@ LoadStreamStub::LoadStreamStub(LoadStreamStub& stub)
 LoadStreamStub::~LoadStreamStub() {
     if (_is_init.load() && !_handler.is_closed()) {
         brpc::StreamClose(_stream_id);
+        auto st = close_wait(60000); // timeout 60s
```

Comment:
why do we need wait in an deconstructor method??
I think we should call wait using a explicit method call and then call dector

Reply:
and 60000 is a magic number